### PR TITLE
added a poem

### PR DESCRIPTION
**Pull request by chriscasto**
I added a poem generated by ChatGPT; fixes issue #4

### CODE REVIEW

**Review by phattya — APPROVED** [review]
Thank you Chris & ChatGPT.  :-P
Lgtm!

File: index.html [hunk 1]
```diff
@@ -11,10 +11,31 @@ <h2>Welcome to this beautiful webpage</h2>
     <img src=https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/23.png />
     <!-- placeholder here to avoid merge conflicts-->
     <!-- placeholder here to avoid merge conflicts-->
-    <p>
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quo, rerum nostrum? Ad deleniti facere in totam
-        officiis? Ea inventore, repudiandae non natus quod illum accusamus maxime ipsum laboriosam impedit. Dolores.
-    </p>
+    <div style="width:375px; margin:25px auto; font-family:Arial">
```
Comment (lisunshiny) [inline index.html:14]: (nit, not in scope of the codelab) -- let's aim to use classes to do styling -- e.g. 

index.html
```html
<div class="poem-container">
  blah
</div>
```
css file
```html
.poem-container {
  width: blah;
  margin: 25px auto;
  ....
}
```

File: index.html [hunk 2]
```diff
@@ -11,10 +11,31 @@ <h2>Welcome to this beautiful webpage</h2>
...
+        <p style="text-align:left;">
+            <span style="font-weight: bold">A poem:</span><br/><br />
```
Comment (lisunshiny) [inline index.html:16]: this poem slaps omg

**Review by lisunshiny — APPROVED** [review]
another non-blocking nit is that git commits are typically written in present tense -- that is, "add a poem" vs. "added a poem"

**Comment by lisunshiny** [comment]
oh, another thing! If you want the issue to close when this PR merges, omit the word "issues" from "fixes issue #blah" . Source: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

File: index.html (hunk 1 above)
Comment (chriscasto) [reply index.html:14]: Totally agree on classes vs inline for "real" code.  I was just being lazy. :)

**Pull request merged**